.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -677,7 +756,7 @@ stages:
 
 - stage: unit_tests_arm64
   condition: and(succeeded(), eq(variables['isBenchmarksOnlyBuild'], 'False'))
-  dependsOn: [build_arm64, master_commit_id]
+  dependsOn: [build_arm64_tracer, master_commit_id]
```

Comment:
Hmmm, just realised we're still not running the native unit tests on Linux/Arm64. We should look into that (subsequent PR)

Reply:
yeah, I got a flashback on the promise I did to run the native unit tests on Linux/Arm64 for Tracer. 
Pin it for a later PR.